src/core_plugins/metrics/public/components/add_delete_buttons.js

```diff
@@ -33,9 +41,13 @@ function AddDeleteButtons(props) {
   if (props.onClone && !props.disableAdd) {
     clone = (
       <Tooltip text={props.cloneTooltip}>
-        <a className="thor__button-outlined-default sm" onClick={props.onClone}>
+        <button
+          aria-label={props.cloneTooltip}
+          className="thor__button-outlined-default sm"
+          onClick={props.onClone}
+        >
           <i className="fa fa-files-o" />
-        </a>
+        </button>
       </Tooltip>
     );
   }
```

Comment:
consider adding tabIndex=0 to the add_delete__buttons. That would likely make tab-order consistent between panels. In the `Annotations` panel they are placed at the end of the tab-order, but in the Data panel they are in-line with the other UX-items.

Reply:
`tabindex=0` will basically just add the element to the tab order in the position related to it's DOM position. A `button` has anyway already an implicit `tabindex` of 0, so it would not change anything. Also you shouldn't use `tabindex`> 0 at all, due to large issues with global ordering. If we find the position of the add/remove button is wrong, we should best move it in the DOM (which would be totally possible, since it is anyway positioned absolut). I found it pretty okay to have them at the end of the series, but I am absolutely open for discussion there.

Reply:
ah ok. didn't realize this. can we move position to the bottom for the other two panels so tab-order is identical?